docs/howto/add-identify.md

```diff
@@ -1,12 +1,114 @@
 # How-to add Identify
 
-Identify is supported on `wms`, `mapserver`, `wfs`, `mapserver-wfs`, and `ags-vector` map-source types.
+If you would like to have your map users to be able to get information on specific features using the Identify tool,
+you must:
+* ensure the `<map-source>` type is one of: `wms`, `mapserver`, `wfs`, `mapserver-wfs`, or `ags-vector`
+* add an identify template to the layer in the `<map-source>` (see below)
+* have the user display the layer in the Catalog
 
-**TL;DR** Add a `<template name="identify">` to the `<layer ...>` which needs identify.
+Add a `<template name="identify">` to the `<layer ...>`.  An easy way to do this is to add
+`<template name="identify" auto="true" />`.  For example, to add identify to the desktop demo's "borders" `<map-source>`
+(titled "City and County Boundaries" in the Catalog), change the `<map-source>` to be:
+
+```xml
+    <map-source name="borders" type="mapserver" title="City and County Borders">
+        <file>./demo/statedata/basemap.map</file>
+        <layer name="city_poly" status="off"/>
+        <layer name="county_borders" status="on">
+            <template name="identify" auto="true" />
+        </layer>
+    </map-source>
+```
+
+When the user turns on the layer in the Catalog, clicks Identify, then clicks inside a county polygon, the details
+for that polygon will be shown in the Super Tab, something like:
+
+```
+City and County Borders
+boundedBy: -10443124.4532125805088.070692-10358662.5561045947016.049725
+AREA: 5163166424.00009
+PERIMETER: 309931.11344
+COUNTY_: 84
+COUNTY_ID: 21
+COUNTY_NUM: 1
+COUNTYNAME: Aitkin
+COUNTYFIPS: 001
+FIPS: 27001
+```
+
+Since the template you just added has `auto="true"`, GeoMoose renders all the feature's properties names and values
+(including the calculated "boundedBy" bounding box for the feature).  The above result is not very pretty, but
+fortunately you can customize the look of the result by writing a custom template for the data.  Instead of an "auto"
+generated template, change the template to be:
+
+```xml
+        <layer name="county_borders" status="on">
+            <template name="identify"><![CDATA[
+            <div>
+                <div class="feature-class county_borders">
+                County
+                </div>
+                <div class="item">
+                    <label>Name:</label>{{properties.COUNTYNAME}}
+                </div>
+            </div>
+            ]]>
+            </template>
+        </layer>
+```
+
+In the example above, there are a few things to note:
+* Customizing the look of the results is done using HTML syntax
+* GeoMoose substitutes values for references it knows about when encounters `{{ }}`
```

Comment:
Is there anything special that needs to be done to escapes/sanitize the data (or avoid escaping/sanitizing the data)?

Reply:
Um, no?

Reply:
Very important points.  I can file a issue and hope the developer responds.